public/pages/IndexDetail/containers/IndexDetail/IndexDetail.tsx

```diff
@@ -0,0 +1,402 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useContext, useEffect, useMemo, useRef, useState } from "react";
+import {
+  EuiTabbedContent,
+  EuiDescriptionList,
+  EuiDescriptionListTitle,
+  EuiDescriptionListDescription,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiFlexGroup,
+  EuiButton,
+  EuiBasicTable,
+  EuiFlyout,
+  EuiFlyoutHeader,
+  EuiTitle,
+  EuiFlyoutBody,
+  EuiIcon,
+  EuiHealth,
+  EuiFormRow,
+  EuiLink,
+} from "@elastic/eui";
+import { get } from "lodash";
+import { Link, RouteComponentProps } from "react-router-dom";
+import { IndexItem } from "../../../../../models/interfaces";
+import IndicesActions from "../../../Indices/containers/IndicesActions";
+import { ManagedCatIndex } from "../../../../../server/models/interfaces";
+import { BREADCRUMBS, IndicesUpdateMode, ROUTES } from "../../../../utils/constants";
+import { ServicesContext } from "../../../../services";
+import { BrowserServices } from "../../../../models/interfaces";
+import IndexFormWrapper, { IndexForm } from "../../../CreateIndex/containers/IndexForm";
+import { HEALTH_TO_COLOR } from "../../../Indices/utils/constants";
+import { CoreServicesContext } from "../../../../components/core_services";
+import { ContentPanel } from "../../../../components/ContentPanel";
+
+export interface IndexDetailModalProps extends RouteComponentProps<{ index: string }> {}
+
+interface IFinalDetail extends ManagedCatIndex, IndexItem {}
+
+const OVERVIEW_DISPLAY_INFO: {
```

Comment:
Move such constants to a `constants` file. It will help keep files small and easier to maintain & review

Reply:
Fixed on https://github.com/opensearch-project/index-management-dashboards-plugin/commit/694dd13ddd334a9f66645a2e5770f2536381af39